workshops.md

```diff
@@ -9,26 +9,40 @@ together to discuss approaches and challenges in implementing scaling
 technologies. Each workshop will be tailored to the member companies attending
 and the specific scaling challenges that they are facing.
 
-- Topics are discussed in a roundtable format in which every participant has an
-  equal opportunity to engage.
+If you have any requests or suggestions for future workshop events, please
+[contact us][optech email].
 
-- Each topic has a moderator and notetaker. The moderator is responsible
-  for a brief introduction of a topic and keeping discussion on track and on
-  time.
+## <span id="taproot-workshop">Workshops #3 & #4 - Schnorr and Taproot Seminars</span>
```

Comment:
Instead of the span, add `{#id-you-want}` after the header definition (this is a Kramdown extension not supported in all Markdown flavors).  E.g.:

    ## Workshops #3 & #4 - Schnorr and Taproot Seminars {#taproot-workshop}

Reply:
Added